packages/paste-core/components/sidebar/src/content-wrappers/SidebarPushContentWrapper.tsx

```diff
@@ -41,7 +41,8 @@ export const SidebarPushContentWrapper = React.forwardRef<HTMLDivElement, Sideba
         {...safelySpreadBoxProps(props)}
         ref={ref}
         element={element}
-        style={breakpointIndex === 0 ? undefined : styles}
+        style={!breakpointIndex ? undefined : styles}
+        marginLeft={['space0', theme.sizes.sizeSidebar]}
```

Comment:
suggestion (non-blocking): can we add some comments here about why we manually apply marginLeft still? 

I also wonder if `!breakpointIndex` is confusing a little. Should we be explicit here so it looks intentional? eg: `breakpointIndex !== undefined && breakpointIndex !== 0`

Reply:
Absolutely!